src/Invite/Invite.model.js

```diff
@@ -0,0 +1,25 @@
+import { Model } from 'decentraland-commons'
+import { SQL } from '../database'
+
+export class Invite extends Model {
+  static tableName = 'decentraland_invite'
```

Comment:
We're using pluralised table names

```suggestion
  static tableName = 'decentraland_invites'
```

Reply:
Ok. I see you are using the plural convention for the table names (im rather use the singular form for many reasons myself). I'll change this.